djangocms_pageadmin/monkeypatch.py

```diff
@@ -40,3 +44,38 @@ def inner(self, **kwargs):
 
 
 VersioningToolbar._add_view_published_button = new_view_published_button(VersioningToolbar._add_view_published_button)
+
+
+def published_date(self, obj):
+    version = StateTracking.objects.filter(version_id=obj.pk)
+    if hasattr(version.first(), "new_state") and version.first().new_state == "published":
+        return version.first().date
+    return ""
+
+
+published_date.short_description = PAGEADMIN_PUBLISHED_DATE_FIELD_LABEL
+admin.VersionAdmin.published_date = published_date
+
+
+def get_list_display(func):
+    """
+    Register the published date field with Versioning Admin
+    """
+    def inner(self, request):
+        original_list_display = func(self, request)
+        list_display_list = list(original_list_display)
+
+        try:
+            created_index = list_display_list.index("created")
+        except ValueError:
+            # if created is missing return the original list to avoid an error
```

Comment:
Could you capitalise the first letter of the comments here, and below, to bring it into line with the rest of the file?